app/src/main/java/org/mozilla/focus/experiments/NimbusSetup.kt

```diff
@@ -124,3 +127,15 @@ fun NimbusException.isReportableError(): Boolean {
         else -> true
     }
 }
+
+/**
+ * Focus specific observer of Nimbus events.
+ *
+ * The generated code `FocusNimbus` provides a cache which should be invalidated
+ * when the experiments recipes are updated.
+ */
+private object EventsObserver : NimbusInterface.Observer {
+    override fun onUpdatesApplied(updated: List<EnrolledExperiment>) {
+        FocusNimbus.invalidateCachedValues()
```

Comment:
This will happen for every cold start of Focus where we invalidate cached values. Is that correct @ionutbedregeanu @Amejia481 ? I would have imagined that we wanted this to happen only on the initial launch of Nimbus.